### PR TITLE
fix editable textarea minimal height

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#17344 ` editable textarea minimal height`](http://community.openproject.org/work_packages/17344)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://0xf013-feature-17344-editable-textarea-minimal-height.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: karma/tests/work_packages/directives/inplace-editor-directive-test.js [hunk 1]
```diff
@@ -105,6 +105,18 @@ describe('inplaceEditor Directive', function() {
         });
         context('wiki_textarea', function() {
           beforeEach(function() {
+            scope.workPackage = {
+              props: {
+                subject: 'Some subject',
+                lockVersion: '1',
+                rawDescription: "1\n2\n\3"
```
Comment (hschink) [inline karma/tests/work_packages/directives/inplace-editor-directive-test.js]: Can you just fix that by using single quotes? Thanks a lot!

File: app/assets/javascripts/angular/ui_components/inplace-editor-directive.js [hunk 2]
```diff
@@ -60,8 +60,19 @@ module.exports = function($timeout, $sce, TextileService) {
     scope.$on('startEditing', function() {
       $timeout(function() {
         element.find('.ined-input-wrapper input, .ined-input-wrapper textarea').focus().triggerHandler('keyup');
+        resizeTextarea(element.find('.ined-input-wrapper textarea').get(0));
       });
     });
+
+    // will move this to textarea-specific components in a separate refactoring pull request
```
Comment (hschink) [inline app/assets/javascripts/angular/ui_components/inplace-editor-directive.js]: You just leave a `TODO` here or (even better) comment it in the commit on GitHub.

**Comment by hschink** [comment]
@0xF013 Just fix the small issues and I'll merge this PR.

File: app/assets/javascripts/angular/ui_components/inplace-editor-directive.js [hunk 3]
```diff
@@ -60,8 +60,19 @@ module.exports = function($timeout, $sce, TextileService) {
...
+    function resizeTextarea(textarea) {
+      if (!textarea) {
+        return;
+      }
+      var lines = textarea.value.split('\n');
```
Comment (myabc) [inline app/assets/javascripts/angular/ui_components/inplace-editor-directive.js]: @0xF013 why don't you just pass in the `jQuery` object and call `val()` on it?.. Then you don't have to worry about existential checks.

Comment (0xF013) [inline app/assets/javascripts/angular/ui_components/inplace-editor-directive.js]: @myabc indeed

File: karma/tests/work_packages/directives/inplace-editor-directive-test.js (hunk 1 above)
Comment (myabc) [inline karma/tests/work_packages/directives/inplace-editor-directive-test.js]: `1\n2\n\3` should read `1\n2\n3`, I think.

Comment (0xF013) [inline karma/tests/work_packages/directives/inplace-editor-directive-test.js]: how can I make linebreaks with single quotes?

Comment (myabc) [inline karma/tests/work_packages/directives/inplace-editor-directive-test.js]: @0xF013 the same way you do with double quotes!

`console.log('1\n2');`

:stuck_out_tongue_winking_eye: 

You're thinking in Ruby!

**Comment by myabc** [comment]
:+1:

**Pull request merged**